pkg/volume/vsphere_volume/vsphere_volume_util.go

```diff
@@ -132,3 +182,50 @@ func getCloudProvider(cloud cloudprovider.Interface) (*vsphere.VSphere, error) {
 	}
 	return vs, nil
 }
+
+// Validate the capability requirement for the user specified policy attributes.
+func validateVSANCapability(capabilityName string, capabilityValue string) bool {
+	switch strings.ToLower(capabilityName) {
```

Comment:
This can be cleaned up. The conversion can be done once and then the validation instead of doing the validation for each case which is duplication.